plugins/filter/jc.py

```diff
@@ -42,6 +42,12 @@
 '''
 
 EXAMPLES = '''
+- name: Install the prereqs of the jc filter (jc python package) so the local Ansible controller
+  delegate_to: localhost
```

Comment:
AFAIUI, we could/should install `jc` on the target node. By explicitly adding this to the example, we might be giving the impression that it is necessary for the module to work.

Reply:
It has to be installed on the controller node, not on the target node.

Reply:
I had failed to recall (or notice) that this is a filter, not a module. In that case, yes, it has to be installed in the controller.

Now, the term _local_ is a bit tricky in that context, because Ansible has the controller and target nodes, but any one of them (or them both) can be local and/or remote. I think it is preferable to avoid those terms (local, remote) and stick to the more Ansible-formal definitions: controller, target.